app/views/application/index.html.erb

```diff
@@ -72,238 +59,7 @@
-                                                      <div class="card"></div>
-                                                      <div class="card"></div>
-                                                      <div class="card"></div>
-                                                      <div class="card"></div>
-                                                      <div class="card"></div>
-                                                      <div class="card"></div>
-                                                      <div class="card"></div>
-                                                      <div class="card"></div>`);
-        for (var i = 0; i < arrayLength; i++) {
-          setContentLoc(i, bird_data_loc[i],bird_images_loc[i],$("#RecentBirdList .scrolling-wrapper .card"));
-        }
-      });
-    }
-
-    function createPos(form){
-      pos = {'coords':{'latitude':parseFloat(form.lat.value),'longitude':parseFloat(form.lng.value)}}
-      parallelHitEbirdEndpoint(pos)
-    }
-
-
-    async function parallelHitEbirdEndpoint(pos, err){
-      if(err) {
-        console.error(err);
-        pos = {
-          coords: {
-            latitude: 37.8039,
-            longitude: -122.2591
-          }
-        };
-      }
-      cur_pos = new L.LatLng(pos.coords.latitude, pos.coords.longitude)
-      mymap.panTo(cur_pos);
-      marker = new L.Marker(cur_pos,{icon: greenIcon});
-      mymap.addLayer(marker);
-      marker.bindPopup(`<b>You are here</b>`).openPopup();
-      hitEbirdEndpoint(pos, false);
-
-    }
-    function hitEbirdEndpoint(pos){
-      var settings = {"url": "/ebird",
-                      "method": "POST",
-                      "timeout": 0,
-                      "data":{
-                          "authenticity_token" : "<%= form_authenticity_token %>",
-                          "lat": pos.coords.latitude,
-                          "lng": pos.coords.longitude,
-                          "num_req":100,
-                          "num_ret":10
-                          }
-                      }
-      $.ajax(settings).done(function (response) {
-        console.log("ebird response", response);
-        markers = []
-        bird_data = response.birddata;
-        bird_images = response.imgsrc;
-        var arrayLength = bird_data.length;
-        for (var i = 0; i < arrayLength; i++) {
-          setContent(bird_data[i],bird_images[i],$("#js-com_bird_info"));
-          markers.push(setMarker(bird_data[i],i))
-        }
-        var group = new L.featureGroup(markers);
-        mymap.fitBounds(group.getBounds().pad(0.2));
-      });
-    }
-
-    getLocation();
-    $( ".j-location_info" ).each(function() {
-      setLoc($(this).data('loc'));
-    });
-  });
-</script>
+<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
+<script src = "app/javascript/channels/index.js" type="text/javascript"></script>
```

Comment:
For consistency, remove spaces around src =

You can also remove type="text/javascript" in modern HTML (from both these attributes), the browser assumes script tags are javascript by default now.